drivers/platform/x86/acer-wmi.c

```diff
@@ -2160,6 +2200,10 @@ static int __init acer_wmi_init(void)
 		interface->capability &= ~ACER_CAP_BRIGHTNESS;
 
 	if (wmi_has_guid(WMID_GUID3)) {
+		if (ACPI_FAILURE(acer_wmi_enable_rf_button())) {
+			pr_err("Cannot enable RF Button Driver\n");
+			return -ENODEV;
```

Comment:
Perhaps we should make this an error but not a failure, in case currently supported devices would fail on this command?